doc/rtd/development/docs_layout.rst

```diff
@@ -0,0 +1,73 @@
+.. _docs_layout:
+
+Documentation directory layout
+******************************
+
+Cloud-init's documentation directory structure, with respect to the root
+directory:
+
+.. code-block:: text
+
+    /doc/
+        - examples/
+        - man/
+        - rtd/
+            - tutorial/
+            - howto/
+            - explanation/
+            - reference/
+
+            - development/
+            - static/
+                - css/
+                - js/
+                - *logos*
+            - *conf.py*
+            - *index.rst*
+            - *links.txt*
+        - rtd_html/
+        - sources/
+
+
+``examples/``
+=============
+
+
+``man/``
```

Comment:
Contains the [Linux man pages](https://en.wikipedia.org/wiki/Man_page) for the binaries provided by cloud-init.